<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewManterPessoa</class>
 <widget class="QWidget" name="ViewManterPessoa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manter pessoa...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nome"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Data de Nascimento:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dtNascimento"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CPF:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idPessoa">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sexo:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="SexoMasculino">
         <property name="text">
          <string>Masculino</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="SexoFeminino">
         <property name="text">
          <string>Feminino</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cpf"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/botaoGravar.png</normaloff>:/botaoGravar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/REMOVER.bmp</normaloff>:/REMOVER.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/fechar.bmp</normaloff>:/fechar.bmp</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>idPessoa</tabstop>
  <tabstop>cpf</tabstop>
  <tabstop>nome</tabstop>
  <tabstop>dtNascimento</tabstop>
  <tabstop>SexoMasculino</tabstop>
  <tabstop>SexoFeminino</tabstop>
  <tabstop>pbSalvar</tabstop>
  <tabstop>pbExcluir</tabstop>
  <tabstop>pbCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
